src/components/ImageMetadata/index.test.tsx

```diff
@@ -30,18 +30,18 @@ describe('the ImageMetadata component', () => {
 
   describe('when no subtitle is passed in the props', () => {
     it('should not include the subtitle span', () => {
-      const wrapper = shallow(<ImageMetadata
+      const wrapper = mount(<ImageMetadata
         caption="aaaaaa"
         credits={{ by: [{ name: 'bbbyyy' }], affiliation: [{ name: 'affff' }] }}
       />);
-      expect(wrapper.find('span')).toHaveLength(0);
+      expect(wrapper.find('span')).toHaveLength(1);
```

Comment:
not sure this test really makes as much sense. I would expect this to still find 0 spans right?

Reply:
Since I decided to use a span to inject HTML with the `dangerouslySetHTML` attribute, we needed a `span` to throw it in. Since there is no subtitle passed to props, but a caption is, the caption is rendered in one `span`. If this test actually had a subtitle, there would be 2 spans. The next test in this suite is the opposite: renders with a subtitle and no caption, so there is only one `span` rendered.

Reply:
but before there were 0 spans found?

Reply:
for caption. if there's no caption, no spans at all should be rendered

Reply:
Oh I see this is for subtitle. Can you find span 0 of the span's class or id, maybe `<span className="title">`

Reply:
Oh I see this is for subtitle. Can you find span 0 of the span's class or id, maybe <span className="title">